core/src/components/tab-button/tab-button.common.scss

```diff
@@ -1,3 +1,5 @@
+@import "../../themes/mixins";
+
```

Comment:
Is this needed? I was able to comment it out and didn't see any use for it.

Reply:
Forgot to remove it when moved things to theme scope... Thank you!